docs/topics/search/indexing.md

```diff
@@ -105,12 +105,12 @@ These are used for performing full-text searches on your models, usually for tex
 
 ### `index.AutocompleteField`
 
-These are used for autocomplete queries which match partial words. For example, a page titled `Hello World!` will be found if the user only types `Hel` into the search box.
+These are used for autocomplete queries that match partial words. For example, a page titled `Hello World!` will be found if the user only types `Hel` into the search box.
 
 This takes the exact same options as `index.SearchField` (with the exception of `partial_match`, which has no effect).
 
 ```{note}
-Only index fields that are displayed in the search results with ``index.AutocompleteField``. This allows users to see any words that were partial-matched on.
+Only index fields are displayed in the search results with ``index.AutocompleteField``. This allows users to see any words that were partial-matched.
```

Comment:
I think "index" is meant to be the verb here - as in, "you should only index these fields".

```suggestion
``index.AutocompleteField`` should only be used on fields that are displayed in the search results. This allows users to see any words that were partial-matched.
```